backend/spec/features/admin/orders/shipments_spec.rb

```diff
@@ -37,6 +37,15 @@
       expect(page).to have_content("shipped package")
       expect(order.reload.shipment_state).to eq("shipped")
     end
+
+    it "doesn't send a shipping confirmation email when ask to suppress the mailer" do
+      find("#send_mailer").set(false)
+      
+      expect {
+        find(".ship-shipment-button").click
+        wait_for_ajax
+      }.not_to change{ ActionMailer::Base.deliveries.count }
```

Comment:
I'm not a big fan of the test.

It's testing with `.not_to` without a corresponding test to check when it does change. For all we know, it could not deliver anything, or this test isn't configured correctly and we wouldn't know it.